builder/ibmcloud/vpc/step_verify_input.go

```diff
@@ -139,6 +142,74 @@ func (s *stepVerifyInput) Run(_ context.Context, state multistep.StateBag) multi
 			state.Put("user_sec_grp_vpc", *secGrp.VPC.ID) // check for vpc is done as part of subnet fetch.
 		}
 	}
+
+	// crn validation
+
+	if config.CatalogOfferingCRN != "" || config.CatalogOfferingVersionCRN != "" || config.EncryptionKeyCRN != "" {
```

Comment:
Can we provide `CatalogOfferingVersionCRN` and `CatalogOfferingCRN` together ?

Reply:
why ? the config validation should happen before input validation